modules/ROOT/pages/salesforce/salesforce-connector-100.adoc

```diff
@@ -0,0 +1,204 @@
+
+This connector works with the Salesforce SOAP API, REST API, Bulk API, and Streaming API, depending on the operation you configure. Each API call uses a request/response pattern over an HTTPS connection. All required request headers, error handling, and HTTPS connection configurations are built into the connector.
+
+== About Connectors
+
+Anypoint Connectors are Mule runtime engine extensions that enable you to connect to APIs and resources on external systems, such as Salesforce, ServiceNow, and Twitter.
+
+== Prerequisites
+
+Before creating an app, you must have access to the Salesforce target resource and Anypoint Platform. You must also understand how to create a Mule app using Design Center or Anypoint Studio, and be familiar with Salesforce, HTTP, SOAP, REST, Bulk and Streaming APIs, Mule runtime engine (Mule), Anypoint Connectors, Anypoint Studio (Studio), Mule concepts, elements in a Mule flow, and Global Elements.
+
+Requirements for using the connector:
+
+* https://developer.salesforce.com[Salesforce developer account]
+* https://help.salesforce.com/articleView?id=user_security_token.htm[Salesforce security token]
++
+You can receive a new security token by email if you run *Reset Security Token* through *My Personal Information* from the Salesforce Setup pages.
+* Consumer key and Secret (available in your Salesforce developer account)
++
+Required if you are using the OAuth Authentication type.
++
+* Namespace and schema location
++
+If you plan to create the XML for your Mule app, you need to include the correct namespace and schema location in your XML file. Anypoint Studio adds this information to the XML file automatically when you add the Salesforce connector to a flow in a Mule app.
+
+Restrictions:
+
+* The Salesforce connector does not expose all possible operations of the Salesforce APIs.
+* The Salesforce connector does not provide access to the Chatter API or the Tooling API.
+
+
+== Audience
+
+* Starting users
++
+Read Get Started (this document) and create your Mule app in
+xref:salesforce/salesforce-connector-design-center.adoc[Design Center] or
+xref:salesforce/salesforce-connector-studio.adoc[Anypoint Studio].
+The xref:salesforce/salesforce-connector-10-config-topics.adoc[Additional Configuration] topic
+helps you extend your understanding of the Salesforce connector.
+* Power users
++
+Read xref:salesforce/salesforce-connector-xml-maven.adoc[XML and Maven Support],
+the xref:salesforce/salesforce-connector-config-topics.adoc[Additional Configuration] topic,
+and xref:salesforce/salesforce-connector-examples.adoc[Examples].
+
+=== Subject Overview
+
+This table provides a quick reference to the topics found in the modules of the Salesforce Connector documentation:
```

Comment:
Consider rewording to indicate that these are links to specific information, rather than "subject" or "topic" or "module" of documentation.

=== Quick Reference Links?
=== Quick Reference Link Table?

"The following table provides quick links to specific Salesforce Connector information:"